templates/test/mocks/describe-cluster/ipv6-cluster.txt

```diff
@@ -0,0 +1 @@
+dGVzdA==	ipv6	http://my-api-endpoint	None	None	None	fe80::1
```

Comment:
Q: Why it would be a txt file?

A: Cause the `aws eks describe-cluster` set `--output` as `text`

https://github.com/awslabs/amazon-eks-ami/blob/v20250519/templates/al2/runtime/bootstrap.sh#L362-L366